docs/Installing.asciidoc

```diff
@@ -177,6 +177,11 @@ Take a look at
 https://registry.opensuse.org/cgi-bin/cooverview?srch_term=project%3Ddevel%3AopenQA[openSUSE's registry]
 for all available container images.
 
+=== Kubernetes
+
+Find a guide and the helm charts for Kubernetes deployment of openQA in
+link:container/helm/README.md[Helm README]
```

Comment:
Paths are treated relative in documents like this one, so this needs `../` to work on GitHub:
```suggestion
Find a guide and the helm charts for Kubernetes deployment of openQA in
link:../container/helm/README.md[Helm README].
```

However, this is also supposed to work from where the documentation is hosted (which is under a completely different domain) so using a relative path won't work there at all. So I think this really needs to be:

```suggestion
Find a guide and the helm charts for Kubernetes deployment of openQA in
link:https://github.com/os-autoinst/openQA/blob/master/container/helm/README.md[Helm README].
```

Note that also the dot at the end was missing in your version.